manager-bundle/tests/HttpKernel/ContaoKernelTest.php

```diff
@@ -312,11 +312,13 @@ public function testSetsRequestTrustedHostsFromEnvVars(): void
     {
         $this->assertSame([], Request::getTrustedHosts());
 
+        $_SERVER['TRUSTED_PROXIES'] = '1.1.1.1,2.2.2.2';
         $_SERVER['TRUSTED_HOSTS'] = '1.1.1.1,2.2.2.2';
```

Comment:
```suggestion
        $_SERVER['TRUSTED_HOSTS'] = '1.1.1.1,2.2.2.2,example.com';
```
This would maybe be helpful for better understanding what “hosts” means in this context?